code/mob/living/carbon/human.dm

```diff
@@ -1176,6 +1176,13 @@
 		var/obj/item/clothing/hider = src.back
 		if (!hider.see_face)
 			. = FALSE
+	for (var/obj/item/hand in list(src.l_hand, src.r_hand))
+		if (istype(hand, /obj/item/paper/newspaper))
+			var/obj/item/paper/newspaper/np = hand
```

Comment:
I don't think you need this cast, `two_handed` is defined on `item` which is what `hand` is typed to anyways

Reply:
yeah im having a lot of bugs with the opening and closing. This part is purely meant to hide the face when two_handed is open. I don't know if it works or not at present.

Reply:
i was playing it safe. Removed the typecast now.